tests/Ontology/AddressTests.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+
+#include "Ontology/Address.h"
+#include "PrivateKey.h"
+#include "PublicKey.h"
+#include "HexCoding.h"
+
+#include <gtest/gtest.h>
+
+using namespace TW;
+using namespace TW::Ontology;
+
+TEST(OntologyAddress, Invalid) {
```

Comment:
```suggestion
TEST(OntologyAddress, Validation) {
```

Not all the asserts are false

Reply:
OK, it's my fault.

Reply:
https://github.com/TrustWallet/wallet-core/pull/103/commits/c6674a786a3319a2963a4b4af36ce704d0c47645